packages/esm-patient-forms-app/src/htmlformentry/html-form-entry.workspace.tsx

```diff
@@ -36,7 +36,11 @@ const HtmlFormEntry: React.FC<HtmlFormEntryComponentProps> = ({
   if (encounterUuid) {
     searchParams.append('encounterId', encounterUuid);
   }
-  searchParams.append('definitionUiResource', htmlForm.formUiResource);
+  if (htmlForm.formUiResource) {
+    searchParams.append('definitionUiResource', htmlForm.formUiResource);
+  } else {
+    searchParams.append('formUuid', htmlForm.formUuid);
+  }
```

Comment:
Really, loading forms from the ui resource isn't something we want to promote IMHO, we should just use the uuid of the form and load the saved form.  To support "development mode", one can set the `definitionUiResource` in the O3 configuration, and that will allow forms to be saved and loaded from configuration files when requested.